calculator.rb

```diff
@@ -0,0 +1,66 @@
+operand = ()
```

Comment:
As a separate note, if operand was something we needed, instead of `()`, it may be good to give `operand` a default value, or set it to `nil`. Maybe something like `operand = 0` or `operand = nil`, just like you did with `operator_select`